Library/Homebrew/formula.rb

```diff
@@ -1258,10 +1258,8 @@ def outdated_kegs(fetch_head: false)
         break
       end
 
-      if current_version
-        []
-      elsif (head_version = latest_head_version) &&
-            !head_version_outdated?(head_version, fetch_head: fetch_head)
+      if current_version ||
+         (latest_head_version && !head_version_outdated?(latest_head_version, fetch_head: fetch_head))
```

Comment:
Are these equivalent or is there another reason for assigning `head_version` to `latest_head_version`?

Reply:
They are functionally equivalent, but calling `latest_head_version` twice will do IO twice.